_posts/2024-01-02-in-progress-wildfly-feature-on-openshift.adoc

```diff
@@ -0,0 +1,318 @@
+---
+layout: post
+title: 'Trying Out An Upcoming Security Feature for WildFly on OpenShift'
+date: 2024-01-02
+tags: oidc authentication OpenShift scopes
+synopsis: How to deploy a wildfly app on OpenShift using an upcoming feature that introduces the ability to add additional scope values when securing applications using OpenID Connect. 
+author: prarthonapaul
+---
+
+:toc: macro
+:toc-title:
+
+A future release of wildfly will include the ability to add additional scope values when securing applications using
```

Comment:
@PrarthonaPaul Total total minor. But everywhere that you refer to WildFly in this blogpost, we can uppercase the W and F